libqtile/confreader.py

```diff
@@ -99,6 +99,20 @@ def update(self, *, fake_screens=None, **settings):
                 value = getattr(self, key, default[key])
             setattr(self, key, value)
 
+    def _reload_config_submodules(self, path):
```

Comment:
Maybe let mypy check this function too? :smile: 
```suggestion
    def _reload_config_submodules(self, path: Path) -> None:
```